### PR TITLE
Adding after failure to nightly + update CI

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

### CODE REVIEW

File: .github/workflows/scripts/install.sh [hunk 1]
```diff
@@ -80,6 +80,8 @@ fi
 
 cat >> vars/main.yaml << VARSYAML
 pulp_settings: {"allowed_export_paths": "/tmp", "allowed_import_paths": "/tmp", "ansible_api_hostname": "http://pulp:80", "ansible_content_hostname": "http://pulp:80/pulp/content"}
+pulp_scheme: http
```
Comment (bmbouter) [inline .github/workflows/scripts/install.sh:83]: is this preparing us for the switch to https?

Comment (fao89) [reply .github/workflows/scripts/install.sh:83]: yep, just switch to `https` and reapply the template

Comment (bmbouter) [reply .github/workflows/scripts/install.sh:83]: niiiiice

**Review by bmbouter — APPROVED** [review]
Thanks!

**Pull request merged**